### ISSUE TITLE
Review and rewrite 'Getting Started with coala'

### ISSUE THREAD

**Issue by ma3lstrom** [coala/documentation#113]
Sentences and sections are hard to follow. See the portion after 

> Let's Start!

**Comment by meetmangukiya** [comment]
Sounds like a relation ? #132

**Comment by Dayfiri** [comment]
I'm confused how in [FAQ](https://github.com/coala/documentation/blob/master/Help/FAQ.rst#id1) there is under `How do I get started with Coala`, a link which leads from FAQ back to same FAQ. What is the purpose of that link then ?

**Comment by sils** [comment]
just browse the docs in http://docs.coala.io/en/latest/Help/FAQ.html

**Comment by abhishekbundela** [comment]
Hey @sils. Assign this issue to me.

**Comment by sils** [comment]
can you try the `cobot assign <issue url>` command on chat for me to see if it works?

**Comment by sils** [comment]
Hi,

please don't ping me personally for such things. Everyone can do them - and almost everyone pings me. If you don't address a person specifically you'll likely get help faster from other people!

(Don't worry, this is not a problem but keeping things general is usually a good policy :))

**Comment by abhishekbundela** [comment]
I added some hyperlinks. Tell me if anything needs to be done.

**Comment by abhishekbundela** [comment]
Please close PR.

**Comment by sils** [comment]
> Please close PR.

What do you want? :/

**Comment by meetmangukiya** [comment]
You can close a PR **you** opened, yourself @abhishekbundela

**Comment by abhishekbundela** [comment]
ohh sorry, I mean please merge PR.

**Comment by abhishekbundela** [comment]
I was thinking something. My mistake.
